### PR TITLE
Httpd: parse backslashes in unquoted directive args

### PR DESCRIPTION
**Pull request by domcleal**
Fixes issue #307

### CODE REVIEW

**Comment by raphink** [comment]
LGTM

File: lenses/httpd.aug [hunk 1]
```diff
@@ -59,7 +59,7 @@ let empty               = Util.empty_dos
 let indent              = Util.indent
 
 (* borrowed from shellvars.aug *)
-let char_arg_dir  = /[^\\ '"\t\r\n]|\\\\"|\\\\'/
+let char_arg_dir  = /([^\\ '"\t\r\n]|[^\\ '"\t\r\n][^ '"\t\r\n]*[^\\ '"\t\r\n])|\\\\"|\\\\'/
```
Comment (pde) [inline lenses/httpd.aug]: I can't review this for correctness yet but I'm going to try to describe what this changes in English.  

Previously, change_arg_dir could be any single non(backslash or "terminator") , or an escaped single or double quote.

Now change_arg_dir could be those things, or any string of two or more non"terminator" chacters, with backslashes permitted in any position except the first and last in the string.

"terminator" means whitespace or quote.

Comment (domcleal) [inline lenses/httpd.aug]: Yep, that's a correct assessment.

**Comment by domcleal** [comment]
Cheers, merged as 155746c.

**Comment by pde** [comment]
Thanks for turning this around so quickly @domcleal!  I think I found a case that this patch didn't catch, and have opened #324 for that.

**Pull request closed without merging**